sdk-cpp/include/do_errors.h

```diff
@@ -11,19 +11,20 @@ namespace microsoft
 {
 namespace deliveryoptimization
 {
-
-enum class errc : int32_t
+namespace errc
 {
-    e_not_impl                  = static_cast<int32_t>(0x80004001), // E_NOTIMPL
-    unexpected                  = static_cast<int32_t>(0x8000FFFF), // E_UNEXPECTED
-    invalid_arg                 = static_cast<int32_t>(0x80070057), // E_INVALIDARG
-    not_found                   = static_cast<int32_t>(0x80070490), // E_NOT_SET (ERROR_NOT_FOUND)
-    no_service                  = static_cast<int32_t>(0x80D01001), // DO_E_NO_SERVICE
-    download_no_progress        = static_cast<int32_t>(0x80D02002), // DO_E_DOWNLOAD_NO_PROGRESS
-    do_e_unknown_property_id    = static_cast<int32_t>(0x80D02011), // DO_E_UNKNOWN_PROPERTY_ID
-    do_e_invalid_state          = static_cast<int32_t>(0x80D02013), // DO_E_INVALID_STATE
-};
 
+constexpr auto e_not_impl                 = static_cast<int32_t>(0x80004001); // E_NOTIMPL
```

Comment:
>e_ [](http://example.com/codeflow?start=15&length=2)

I vote for removing the 'e_' and 'do_e_' prefixes since they are already inside an errc namespace.

Reply:
(prefixes were not required inside the enum either)

Reply:
Agreed